contentcuration/contentcuration/utils/files.py

```diff
@@ -221,6 +225,17 @@ def generate_thumbnail_from_node(node, set_node=None):  # noqa
     return thumbnail_object
 
 
+def generate_thumbnail(item, dimension=200):
```

Comment:
One other minor suggestion. Might be good to name this `generate_thumbnail_from_channel`, as I initially wondered if this could be called for both channel and ContentNode thumbnail generation, but icon_encoding does not exist on ContentNode so this will only work for channels.

Reply:
What if I add an extra check like `hasattr(item, 'icon_encoding')` as the rest of the code should work on content nodes (once we move nodes to jsonfields as well)?

Reply:
Actually, it looks like this isn't even used. Going to remove it for now

Reply:
Actually, I see a generate thumbnail from node method. Updating the name!